lib/showoff.rb

```diff
@@ -407,11 +408,8 @@ def process_markdown(name, section, content, opts={:static=>false, :pdf=>false,
 
         # Apply the template to the slide and replace the key to generate the content of the slide
         sl = process_content_for_replacements(template.gsub(/~~~CONTENT~~~/, slide.text))
-        sl = Tilt[:markdown].new(nil, nil, engine_options) { sl }.render
```

Comment:
Mmm, if we're making a delineation between pre & post markdown, I think I'd rather have either a method called render_markdown, or have the Tilt call inline.